tests/console/journalctl.pm

```diff
@@ -128,7 +128,8 @@ sub run {
         script_run("journalctl --boot=-1 | grep 'no persistent journal was found'") or die "Persistent journal should not be enabled by default on SLE!\n";
         assert_script_run "mkdir ${\ PERSISTENT_LOG_DIR }";
         assert_script_run "systemd-tmpfiles --create --prefix ${\ PERSISTENT_LOG_DIR }";
-        # test for imuxsock
+        # test for installed rsyslog and for imuxsock existance
+        assert_script_run 'rpm -q rsyslog';
```

Comment:
Perhaps add a comment like `# rsyslog must be there by design` ?

Reply:
updated